docs/conf.py

```diff
@@ -49,7 +49,7 @@
 ##############################################################################
 
 baseBranch = "master"
-useGitHubURL = False
+useGitHubURL = True
 commitSHA = getenv('GITHUB_SHA')
 githubBaseURL = "https://github.com/intel/cri-resource-manager/"
```

Comment:
I think the default base URL should be possible to override, for instance from the workflow to get it
automatically working correctly in forks as well.